src/runtype.ts

```diff
@@ -183,3 +183,9 @@ export function isFail(v: unknown): v is Fail {
 
   return (v as any)[failSymbol]
 }
+
+export function isNonStrictRuntype(fn: Runtype<any>): boolean {
+  return !!(fn as any).isNonStrict
+}
+
+export const isNonStrictRuntypeSymbol = Symbol('isNonStrict')
```

Comment:
Not sure if these should be here or in `src/nonStrict.ts`, or maybe somewhere else?